source-code/design-system/src/color-system/generateTokens.cts

```diff
@@ -22,19 +22,19 @@ export function generateTokens(config: Config): ColorTokens {
 		background: white,
 		"on-background": neutralColors.neutral[900],
 		"surface-100": new TinyColor(neutralColors.neutral[900])
-			.lighten(88)
+			.lighten(surface100Color)
 			.toHex8String(),
 		"surface-200": new TinyColor(neutralColors.neutral[900])
-			.lighten(85)
+			.lighten(surface100Color - 3)
```

Comment:
The Material spec does not jump in conistent steps. Read https://m3.material.io/styles/color/the-color-system/color-roles. I am going ahead and fix that